cdap-app-fabric/src/test/java/io/cdap/cdap/internal/app/services/ApplicationLifecycleServiceTest.java

```diff
@@ -128,6 +148,34 @@ public void testAppDeletionMultipleNS() throws Exception {
     deleteAppAndData(testNSAppId);
   }
 
+  @Test
+  public void testAcceleratorMetaDataDeletion() throws Exception {
+    Id.Artifact artifactId = Id.Artifact
+      .from(Id.Namespace.DEFAULT, AppWithWorkflow.class.getSimpleName(), "1.0.0-SNAPSHOT");
+    Location appJar = AppJarHelper.createDeploymentJar(locationFactory, AppWithWorkflow.class);
+    File appJarFile = new File(tmpFolder.newFolder(),
+                               String.format("%s-%s.jar", artifactId.getName(), artifactId.getVersion().getVersion()));
+    Files.copy(Locations.newInputSupplier(appJar), appJarFile);
+    appJar.delete();
+
+    //deploy app
+    applicationLifecycleService
+      .deployAppAndArtifact(NamespaceId.DEFAULT, AppWithWorkflow.class.getSimpleName(), artifactId, appJarFile, null,
+                            null, programId -> {
```

Comment:
Have it on one line. `programId -> { }`

Reply:
Done. Have to fix the intellij settings.